src/customFileParser/CustomFileReader.ts

```diff
@@ -108,8 +108,30 @@ export default class CustomFileReader {
     return loadUncompressedValues(this);
   }
 
-  readSectorGeometryIndexHandlers(sectorEndLocation: number): GeometryIndexHandler[] {
-    return loadGeometryIndexHandlers(this, sectorEndLocation);
+  readCompressedGeometryData(sectorEndLocation: number): {
+    primitives: CompressedGeometryData[],
+    instancedMesh: CompressedGeometryData | undefined,
```

Comment:
You can instead do (I think)
`instancedMesh?: CompressedGeometryData,`

Reply:
Done